homework2_count_vow.py

```diff
@@ -0,0 +1,73 @@
+# Task description:
+# The English language has five vowels: A, E, I, O, and U
+# Please count each vowel occurence in text bellow
+# ( sum of lower and capital cases)

```

Comment:
Still present linter errors
./T-HOMETASKS/L02/000.py:1:1: D100 Missing docstring in public module
./T-HOMETASKS/L02/000.py:24:38: W291 trailing whitespace
./T-HOMETASKS/L02/000.py:34:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:36:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:47:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:50:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:50:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:51:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:51:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:52:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:52:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:53:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:53:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:54:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:54:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:55:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:55:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:56:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:56:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:57:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:57:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:58:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:58:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:59:5: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:59:10: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:59:13: C812 missing trailing comma
./T-HOMETASKS/L02/000.py:62:1: D103 Missing docstring in public function
./T-HOMETASKS/L02/000.py:62:1: E302 expected 2 blank lines, found 1
./T-HOMETASKS/L02/000.py:63:21: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/000.py:73:21: W292 no newline at end of file